chef/cookbooks/crowbar/templates/default/crowbar-shutdown-services-before-upgrade.sh.erb

```diff
@@ -64,4 +72,13 @@ for service in cinder-volume cinder-scheduler; do
     done
 done
 
+ret=$?
+if [ $ret != 0 ] ; then
+    echo "Error occured during deleting cinder services"
+    echo $ret > $UPGRADEDIR/crowbar-shutdown-services-before-upgrade-failed
```

Comment:
I've fixed it here, but we have to make sure it is deleted in other scripts as well: https://trello.com/c/QrxUXWGe/212-ensure-the-failed-files-are-deleted-when-upgrade-bash-scripts-are-starting